encord/user_client.py

```diff
@@ -265,12 +270,30 @@ def create_project(
         dataset_hashes: List[str],
         project_description: str = "",
         ontology_hash: str = "",
-        source_projects: List = list(),
```

Comment:
that's not backwards compatible. Can we remove the order shift? 
You can also add a `*` to have keyword only arguments.